Source/WebCore/platform/graphics/cg/ShareableBitmapCG.mm

```diff
@@ -129,28 +128,26 @@
     if (!pixels)
         return nullptr;
 
-    const auto* bytes = byteCast<uint8_t>(CFDataGetBytePtr(pixels.get()));
-    CheckedUint32 sizeInBytes = CFDataGetLength(pixels.get());
-    if (!bytes || !sizeInBytes || sizeInBytes.hasOverflowed())
+    auto bytes = WTF::span(pixels.get());
+    if (bytes.empty() || CheckedUint32(bytes.size()).hasOverflowed())
         return nullptr;
```

Comment:
Do you understand where the "must not overflow 32 bits" requirement comes from?

Reply:
I do not, I'll try and figure it out. ShareableBitmapConfiguration seems to be consistently using uint32_t (or checked versions) everywhere.